.ci/test.sh

```diff
@@ -69,7 +69,7 @@ if [[ $TASK == "if-else" ]]; then
     exit 0
 fi
 
-if [[ $TASK == "r-package" ]]; then
+if [[ $TASK == "r-package" ]] || [[ $TASK == "check-r-docs" ]]; then
```

Comment:
Just a suggestion to not overcomplicate this `if` statement in the future. Name task with something like `r-package-check-docs` and then just check that it starts with `r-package`.

Reply:
oh good idea! Just pushed https://github.com/microsoft/LightGBM/pull/3205/commits/0b17315310b2cf2f97a19fb9c907dcd8eab4e86b to implement this